core/src/main.cpp

```diff
@@ -288,6 +295,12 @@ constexpr auto default_jsonschema() -> std::string_view
                                     "required": [
                                         "user_attribute_mapping"
                                     ]
+                                }},
```

Comment:
Instead of using `user_attribute_mapping`, users would simply have to use the 'local file' user mapping plugin. This would require the two new config options to be used. What went into `user_attribute_mapping` would then be moved into a separate JSON file.

Reply:
Removed.